lib/taskana-core/src/main/java/pro/taskana/spi/task/api/ReviewRequiredProvider.java

```diff
@@ -0,0 +1,41 @@
+package pro.taskana.spi.task.api;
+
+import pro.taskana.common.api.TaskanaEngine;
+import pro.taskana.task.api.models.Task;
+
+/**
+ * The ReviewRequiredProvider allows to implement customized behaviour which requests a review
+ * instead of completing a {@linkplain Task}.
+ */
+public interface ReviewRequiredProvider {
+
+  /**
+   * Provide the active {@linkplain TaskanaEngine} which is initialized for this TASKANA
+   * installation.
+   *
+   * <p>This method is called during TASKANA startup and allows the service provider to store the
+   * active {@linkplain TaskanaEngine} for later usage.
+   *
+   * @param taskanaEngine the active {@linkplain TaskanaEngine} which is initialized for this
+   *     installation.
+   */
+  default void initialize(TaskanaEngine taskanaEngine) {}
+
+  /**
+   * Determine if a {@linkplain Task} has to be reviewed instead of completed before {@linkplain
+   * pro.taskana.task.api.TaskService#completeTask(String)} is executed.
+   *
+   * <p>This SPI is executed with the same {@linkplain
+   * pro.taskana.common.api.security.UserPrincipal} and {@linkplain
+   * pro.taskana.common.api.security.GroupPrincipal} as in {@linkplain
+   * pro.taskana.task.api.TaskService#completeTask(String)}.
+   *
+   * @param task the {@linkplain Task} before {@linkplain
+   *     pro.taskana.task.api.TaskService#completeTask(String)} has started
+   * @return true, if {@linkplain pro.taskana.task.api.TaskService#requestReview(String)} should be
+   *     executed instead of {@linkplain pro.taskana.task.api.TaskService#completeTask(String)}.
+   *     False, if {@linkplain pro.taskana.task.api.TaskService#completeTask(String)} can be
+   *     executed regularly.
```

Comment:
Please remove the points at the end according to our guidelines

Reply:
done